### DISCUSSION TITLE
Appcenter alternate options discusiion

### DISCUSSION THREAD

**Discussion by akhilvswoodplc** [CommunityToolkit/Maui#2270] [Q&A] [answered]
Hello, 
I am in reasearch for Appcenter alternate option for app distribution, crash reporting and user analytics. The difficult part is it is difficult to find one solution which supports all the platforms like iOS, Android and Windows.

Firebase had bothdistribution and Crashlytics, but it lacks support for Windows.
Anyone have experience here?

**Comment by TheCodeTraveler** [comment, marked as answer]
I recommend https://sentry.io for crash + analytic reporting. 

For app distribution, use the platforms' built-in systems, Test Flight and Google Play.

**Comment by CliffAgius** [comment]
Closed as answered
